editor/post-title/index.js

```diff
@@ -0,0 +1,43 @@
+/**
+ * External dependencies
+ */
+import { connect } from 'react-redux';
+
+/**
+ * Internal dependencies
+ */
+import './style.scss';
+
+function PostTitle( { title, onUpdate } ) {
+	const onChange = ( event ) => onUpdate( event.target.value );
+
+	return (
+		<h1 className="editor-post-title">
```

Comment:
I guess it makes sense in the context of being the `<h1 />` of the post itself (title). I think my concern is in the context of the admin screen. Not really sure what the correct answer is here.